schema/resolversTests/validation.test.js

```diff
@@ -211,6 +211,30 @@ describe("resolvers", () => {
         inclusive: true
       });
     });
+
+    it("can update inclusive and previous answer max values", async () => {
+      const previousAnswer = await createNewAnswer(firstPage, "Number");
+      const currencyAnswer = await createNewAnswer(firstPage, "Currency");
+      const currencyValidation = await queryAnswerValidations(
+        currencyAnswer.id
+      );
+
+      const result = await mutateValidationParameters({
```

Comment:
Does `entityType` not need to be passed?

Reply:
Just added separate tests for entityType.  It is updated independently.